src/main/java/net/milkbowl/vault2/chat/Chat.java

```diff
@@ -0,0 +1,995 @@
+/* This file is part of Vault.

```

Comment:
I know this was mentioned within the PR itself, but we definitely should be updating both Chat and Permission in the same fashion as Economy. We can do this now or we can do so in another PR, either is fine by me.

Reply:
I would like to keep this PR concise and limited to just Economy. I think the code related to Chat and Permissions should be improved by chat and permission plugin authors.